jax/random.py

```diff
@@ -530,6 +530,50 @@ def _shuffle(key, x, axis):
   return x
 
 
+def choice(key, a, size=(), replace=True, p=None):
```

Comment:
```suggestion
def choice(key, a, shape=(), replace=True, p=None):
```

I think using `shape` rather than `size` is more consistent with the rest of the file.